apcd-cms/src/client/src/components/Submitter/Extensions/ExtensionsForm.tsx

```diff
@@ -0,0 +1,273 @@
+                  </Col>
+
+                  <Col md={4}>
+                    <FormGroup className={`position-relative ${styles.dateInputContainer}`}>
+                      <Label for={`extensions.${index}.requestedTargetDate`}>
+                          Requested Target Date <sup>2</sup>
+                          <span className={styles.requiredText}>{" "}(required)</span>
+                      </Label>
+                      <Field
+                          type="date"
+                          name={`extensions.${index}.requestedTargetDate`}
+                          id={`requestedTargetDate_${index}`}
+                          className={`form-control ${styles.dateInputField}`}
+                      />
+                      <ErrorMessage name={`extensions.${index}.requestedTargetDate`} component={FormFeedback} />
+                    </FormGroup>
+                  </Col>
+
+                  <Col md={4}>
+                    <FormGroup className={`position-relative ${styles.dateInputContainer}`}>
+                      <Label for={`extensions.${index}.currentExpectedDate`}>
+                        Current Expected Date <sup>3</sup>
+                        <span className={styles.requiredText}>{" "}(required)</span>
+                      </Label>
+                      <Field
+                        type="date"
+                        name={`extensions.${index}.currentExpectedDate`}
+                        className="form-control"
+                      />
+                      <ErrorMessage name={`extensions.${index}.currentExpectedDate`} component={FormFeedback} />
+                    </FormGroup>
+                  </Col>
+                </Row>
+
+                <hr />
+              </div>
+            ))}
+
+            <Button className="c-button c-button--primary"
+              type="button"
+              onClick={() =>
+                setFieldValue('extensions', [
+                  ...values.extensions,
+                  {
+                    businessName: '',
+                    extensionType: '',
+                    applicableDataPeriod: '',
+                    requestedTargetDate: '',
+                    currentExpectedDate: '',
+                  },
+                ])
+              }
+              color="primary"
+            >
+              + Add Another Extension Request
+            </Button>
+            {" "}
+            <Button className="c-button c-button--secondary"
+              type="button"
+              onClick={() =>
+                values.extensions.length > 1 &&
+                setFieldValue('extensions', values.extensions.slice(0, -1))
+              }
+              color="secondary"
+              disabled={values.extensions.length === 1}
+            >
+              - Remove Last Extension Request
+            </Button >
+
+            <hr />
+            <h4>Request and Justification</h4>
+            <FormGroup>
+              <Label for="justification">
```

Comment:
for accessibility - htmlFor is suggested instead of for
```suggestion
              <Label htmlFor="justification">
```